packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -180,36 +188,63 @@ class ProxyController {
      * @param {HTTP_VERB} method
      * @param {ApplicationConstructedProxyConfiguration} configBody
      */
-    private sendToHttpMethod(
-        res: Response,
-        next: NextFunction,
-        method: HTTP_VERB,
-        configBody: ApplicationConstructedProxyConfiguration,
-        activityLogId: number,
-        environment_id: number,
-        isSync?: boolean,
-        isDryRun?: boolean
-    ) {
+    private sendToHttpMethod({
+        res,
+        method,
+        configBody,
+        activityLogId,
+        environment_id,
+        isSync,
+        isDryRun
+    }: {
+        res: Response;
+        method: HTTP_VERB;
+        configBody: ApplicationConstructedProxyConfiguration;
+        activityLogId: number;
+        environment_id: number;
+        isSync?: boolean | undefined;
```

Comment:
do we the undefined when using `?`

Reply:
when using strict mode typescript makes a distinction between a inexistant value `?` vs a value pointing to `undefined`. It enforces object shape but can be annoying